src/core_plugins/spy_modes/public/table_spy_mode.js

```diff
@@ -1,10 +1,9 @@
 // import 'ui/agg_table';
-import { AggResponseTabifyProvider } from 'ui/agg_response/tabify/tabify';
+import { tabifyAggResponse } from 'ui/agg_response/tabify/tabify';
```

Comment:
in every file where used we import tabifyAggResponse directly instead its provider 
- we no longer need to call Private to get tabifyAggResponse
- we pass response aggs into tabifyAggResponse (instead of vis)
-- we also need to pass isHierarchical as option (instead of using vis.isHierarchical inside tabify)